examples/http-data-binding/http_data_binding.description

```diff
@@ -1,3 +1,3 @@
 // HTTP data binding helps access payload through the last resource signature parameter. The parameter name should be
-// declared in the resource config under the 'body' annotation. `string`, `json`, `xml`, `byte[]`, and records are
+// declared in the resource config under the `body` annotation. `string`, `json`, `xml`, `byte[]`, and records are
```

Comment:
IIRC, only json is supported now.

Reply:
No, the description is correct. Now data binding supports record[] as well. Shall we add that too?